flask_httpauth.py

```diff
@@ -62,15 +63,16 @@ def authenticate_header(self):
 
     def get_auth(self):
         auth = request.authorization
-        if auth is None and 'Authorization' in request.headers:
+        if (auth is None and "Authorization" in request.headers)\
+                or self.header in request.headers:
```

Comment:
I don't think this resolves the problem I pointed at earlier. If you send a token in your custom header, and a valid basic or digest auth header in `Authorization`, the basic/digest will be preferred in all cases, including when you use token auth and pass the custom header as your choice.

Reply:
My bad, I thought we wanted the custom header to take precedence over the default if provided 🤦‍♂️ Thanks for the example, I've just fixed it.